### ISSUE TITLE
SqlStatementMetaData.addTable() needs to compare against table name in upper case

### ISSUE THREAD

**Issue by tmielke** [syndesisio/syndesis#1346]
`SqlStatementMetaData.addTable()` currently reads: 

    public String addTable(String tableName) throws SQLException {
        if (tablesInSchema.contains(tableName)) {
            tableNames.add(tableName);
        } else {
            throw new SQLException("Table does not exist in schema " + schema);
        }
        return tableName;
    }

but the call to 

    if (tablesInSchema.contains(tableName)) 

should rather be 

    if (tablesInSchema.contains(tableName.toUpperCase())) 

because tables added to `tablesInSchema` are all converted to upper case in 
`DatabaseMetaDataHelper.fetchTables()`:

    tablesInSchema.add(rs.getString(3).toUpperCase());

As a result, trying to add an SQL statement that does not use an all uppercase table name in the "Invoke SQL" step will fail silently. While I can click the Done button in the Syndesis UI nothing happens but the syndesis-verifier pod raises

    Caused by: java.sql.SQLException: Table does not exist in schema null
	at io.syndesis.connector.sql.SqlStatementMetaData.addTable(SqlStatementMetaData.java:48) ~[sql-common-1.2-SNAPSHOT.jar!/:1.2-SNAPSHOT]
	at io.syndesis.connector.sql.SqlStatementParser.parseInsert(SqlStatementParser.java:97) ~[sql-common-1.2-SNAPSHOT.jar!/:1.2-SNAPSHOT]
	at io.syndesis.connector.sql.SqlStatementParser.parse(SqlStatementParser.java:78) ~[sql-common-1.2-SNAPSHOT.jar!/:1.2-SNAPSHOT]

**Comment by zregvart** [comment]
@KurtStam mind having a look?

**Comment by KurtStam** [comment]
@tmielke what database are you using?

**Comment by tmielke** [comment]
@KurtStam I used the embedded Postgres DB that comes out of the box in syndesis-db pod with a connection to it pre-defined in the Syndesis UI.
I created a new table in this existing sampledb using SQL syntax like

    CREATE TABLE Test VALUES (...);

The table name was not in all capital letters. 
OTOH it should not make much difference what DB is used due to the code in `DatabaseMetaDataHelper.fetchTables():`

    tablesInSchema.add(rs.getString(3).toUpperCase());

which seems to turn all table names into upper case strings.
